src/perf/components/PageSpinner.tsx

```diff
@@ -7,7 +7,7 @@ import {
 
 interface Props {
   loading?: RemoteDataState
-  children?: React.ReactChild
+  children?: any
```

Comment:
aw man - can we avoid this change?

Reply:
try revert it and see what breaks. i think it should be fine. looks like it might have been a ReactChild | ReactChild[] issue that isn't needed anymore